src/sunlinsol/band/fmod/fsunlinsol_band_mod.f90

```diff
@@ -20,9 +20,10 @@
 
 module fsunlinsol_band_mod
  use, intrinsic :: ISO_C_BINDING
- use fsundials_linearsolver_mod
  use fsundials_types_mod
+ use fsundials_linearsolver_mod
  use fsundials_context_mod
+ use fsundials_types_mod
```

Comment:
Not sure why but `use fsundials_types_mod` is in here twice

Reply:
Looks like that's true for the other `.f90` files as well

Reply:
This is not a new issue and I currently don't know of a fix for it. It has to do with our swig module structure. It seems to be harmless though.